pysat/utils/io.py

```diff
@@ -1725,7 +1725,8 @@ def inst_to_netcdf(inst, fname, base_instrument=None, epoch_name=None,
     # Remove any attributes with the names below. pysat is responsible
     # for including them in the file.
     pysat_items = ['Date_End', 'Date_Start', 'File', 'File_Date',
-                   'Generation_Date', 'Logical_File_ID']
+                   'Generation_Date', 'Logical_File_ID', 'acknowledgements',
```

Comment:
Wanted to bring this question into the pull request.
Do these items still need to be popped?
We're assigning all of them in the following lines, so maybe they can be left un-popped.

Reply:
I haven't tried swapping these out.  Not sure if this is a remnant from python 2.7.  acknowledgements and references have to be popped for compatibility with the operational bits below.